oak_restricted_kernel/src/interrupts.rs

```diff
@@ -286,12 +296,27 @@ extern "x86-interrupt" fn machine_check_handler(stack_frame: InterruptStackFrame
     shutdown::shutdown();
 }
 
+bitflags! {
+    #[repr(transparent)]
+    #[derive(Debug)]
+    pub struct Mxcsr: u32 {
+        const DE = _MM_EXCEPT_DENORM;
+        const ZE = _MM_EXCEPT_DIV_ZERO;
+        const PE = _MM_EXCEPT_INEXACT;
+        const IE = _MM_EXCEPT_INVALID;
+        const OE = _MM_EXCEPT_OVERFLOW;
+        const UE = _MM_EXCEPT_UNDERFLOW;
+    }
+}
 extern "x86-interrupt" fn simd_fp_handler(stack_frame: InterruptStackFrame) {
     error!("KERNEL PANIC: SIMD FLOATING POINT EXCEPTION!");
     error!(
         "Instruction pointer: {:#016x}",
         stack_frame.deref().instruction_pointer.as_u64()
     );
+    let mxcsr = Mxcsr::from_bits_retain(unsafe { _MM_GET_EXCEPTION_STATE() });
```

Comment:
Even though we are about to crash, perhaps add a safety comment for consistency.